public/javascripts/Gallery/src/modal/Gallery.Panorama.js

```diff
@@ -0,0 +1,283 @@
+/**
+ * A holder class that inserts a GSV Pano into the supplied DOM element
+ * 
+ * @param {HTMLElement} svHolder The DOM element that the GSV Pano will be placed in
+ * @returns {GalleryPanorama} The gallery panorama that was generated
+ */
+ function GalleryPanorama(svHolder) {
+    var self = {
+        className: "GalleryPanorama",
+        label: undefined,
+        labelMarkers: [],
```

Comment:
It looks like you're storing all the label markers that you create in here, but you aren't reusing them either. Not sure that we even need to keep them at all (I guess it depends on how much work it takes to create one). But if I open and close the same modal, I think we are just adding more copies of the same label to this array.